src/fs/patch.ts

```diff
@@ -70,8 +70,8 @@ function shimFs(binary: NexeBinary, fs: any = require('fs')) {
       ino: 0,
       nlink: 0,
       rdev: 0,
-      uid: 123,
-      gid: 500,
+      uid: process.env.NEXE_UID ?? 123,
+      gid: process.env.NEXE_GID ?? 500,
```

Comment:
Yeah, These properties shouldn't be here... Sorry for this issue. I'll send another PR.